arwen/elrondapi/elrondei.go

```diff
@@ -2442,6 +2713,13 @@ func v1_4_executeOnDestContext(
 	dataOffset int32,
 ) int32 {
 	host := arwen.GetVMHost(context)
+	runtime := host.Runtime()
+	metering := host.Metering()
+	initialGasLeft := metering.GasLeft()
+	defer func() {
```

Comment:
won't this include all the gas consumed within the called contract?
I guess it's fine for the first version, but we should think about this